src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/service/prison/AddressService.kt

```diff
@@ -88,7 +92,10 @@ class AddressService(
         ?: throw EntityNotFoundException("Cannot find standard address check request with id: $requestId")
 
     if (standardAddressCheckRequest.assessment.offender.prisonNumber != prisonNumber) {
-      throw HttpClientErrorException(HttpStatus.NOT_FOUND, "Standard address check request id: $requestId is not linked to offender with prison number: $prisonNumber")
+      throw HttpClientErrorException(
```

Comment:
we use an EntityNotFound exception in a few places already?

Reply:
ahh yes, I'll update